ports/unix/Makefile

```diff
@@ -270,13 +270,16 @@ endif
 
 deplibs: libffi axtls
 
+libffi: $(BUILD)/libffi/include/ffi.h
+
+$(TOP)/lib/libffi/configure: $(TOP)/lib/libffi/autogen.sh
+	cd $(TOP)/lib/libffi; ./autogen.sh
```

Comment:
This no longer does a `git clean` like it did before.  Is that intentional?

Reply:
> This no longer does a git clean like it did before. Is that intentional?

Sure, that's the whole idea. It built inside source dir, so between builds, had to reset it to clean-room state to avoid one build's config to leak into another, with git clean a surely robust way to achieve it.

As builds now happen in dedicated build dirs, the src stays clean.